pages/[wiki]/[article]/index.tsx

```diff
@@ -0,0 +1,145 @@
+import fs from 'fs';
```

Comment:
Why `[wiki]/[article]/index.tsx` and not `wiki/[article].tsx` as recommended by Next.js documentation? 
https://nextjs.org/docs/routing/dynamic-routes

Why `[wiki]` and not just `wiki`? Does it need to be dynamic? Won't it always be `movapp.cz/wiki/...` ?

Reply:
wiki page is different for each locale. so I need to generate one page for movapp/wiki and different page for movapp/uk/wiki. That's why I used dynamic route for wiki page and defined different page for both uk and cs locales.
Is there any way to do it differently?

Reply:
Every page is different for each locale. For example `alphabet` becomes `uk/alphabet` in the uk locale and renders different content based on that locale. Why does not work here? The locale prefix should be handled automatically by next.js, not manually.

Reply:
If we use just `wiki` then we don't need `getStaticPaths` and only one version of content will be available. We will have the same content for both `cs` and `uk`. `uk/alphabet` is different from `alphabet` because we conditionally define content based on locale. 
Does it make sense or I missed something?